src/user/views/author_views.py

```diff
@@ -87,6 +88,47 @@ def update(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @action(detail=True, methods=["get"], permission_classes=[AllowAny])
+    def summary_stats(self, request, pk=None):
+        author = self.get_object()
+        cache_key = f"author-{author.id}-summary-stats"
+        cache_hit = cache.get(cache_key)
+
+        if cache_hit:
+            return Response(cache_hit)
```

Comment:
Nit: Just to align with the code further below in the method.
```suggestion
            return Response(cache_hit, status=200)
```